File: examples/ai-review/src/sample.ts
export function sum(a: number, b: number) {
  return a + b;
}

const runUnsafe = (cmd: string) => {
  // intentionally naive example
  return cmd;
}

export class Person {
  constructor(public name: string) {}
  greet() { return `Hi ${this.name}`; }
}

File: examples/ai-review/src/sample.py
import subprocess
from typing import Optional

def run(cmd: str, timeout: Optional[int] = None) -> str:
    # intentionally naive example
    result = subprocess.check_output(cmd, shell=True, timeout=timeout)
    return result.decode("utf-8")

class User:
    def __init__(self, name: str):
        self.name = name

    def greeting(self) -> str:
        return f"Hello, {self.name}"
